process/banning.html

```diff
@@ -55,7 +55,7 @@ <h4>Nearby</h4>
 <li>from a single group that the individual has not formally joined, to the chair(s) of the group; and</li>
 <li>from all W3C groups, to the CEO and COO jointly.</li>
 </ul>
-<p>If the chair, team contact, CEO, COO are in any way involved as accused or accuser, they should recuse themselves from all discussions.</p>
+<p>If the chair, team contact, CEO, COO are involved as accused or accuser, they should recuse themselves from all discussions. For clause (2) the escalation is to another chair and may involve a <a href="https://www.w3.org/Consortium/pwe/#ombuds">W3C Ombuds</a> then to the CEO. For clause (3), a request to remove the CEO or COO from a group is not one that ought to be delegated. For clause (4), in extremis involving the CEO or COO, either should be able to act without the other.</p>
 
```

Comment:
(I know my review was not requested, but I'd opt for the simpler edit: s/in any way// )

Documents that are too prescriptive can end up with weird corner cases.

Reply:
(I had requested your review a couple days ago but removed it moments ago, not knowing whether you wanted to further review!)

I removed "in any way" in the latest commit that adds Ralph's escalation wording.

Reply:
and also remove the escalation wording.

Reply:
#97